commons/src/main/java/io/aiven/kafka/connect/common/source/input/ParquetTransformer.java

```diff
@@ -46,6 +48,7 @@
 
 public class ParquetTransformer implements Transformer {
 
+    private static final AvroData AVRO_DATA = new AvroData(100);
```

Comment:
same global cache concern as the JsonConverter.

Reply:
Moved this instantiation to TransformerFactory too. So same cache  will be used throughout the application making it consistent I believe